clang/include/clang/Driver/Options.td

```diff
@@ -3929,6 +3929,10 @@ def foperator_arrow_depth_EQ : Joined<["-"], "foperator-arrow-depth=">, Group<f_
   Visibility<[ClangOption, CC1Option]>,
   HelpText<"Maximum number of 'operator->'s to call for a member access">,
   MarshallingInfoInt<LangOpts<"ArrowDepth">, "256">;
+def fsycl_emit_external_funcs_as_entry_points : Flag<["-"], "fsycl-emit-external-funcs-as-entry-points">,
+  Group<sycl_Group>, HelpText<"Treat `SYCL_EXTERNAL` functions as entry points in sycl-post-link tool">;
```

Comment:
I wonder if we need to be more user-friendly here: not everyone knows what `sycl-post-link` is and therefore this description won't help to understand the effect of the option.

I don't have a suggestion how to rewrite it completely, but we could just add something like: "(prevents removal of unused `SYCL_EXTERNAL` functions, making them available for some advanced interoperability use cases)".

Reply:
This is a good point - the `sycl-post-link` tool is not something users know about.  Just noting about what the option does should be sufficient without calling to what tool takes care of the behavior.

Reply:
Thanks very much for the feedback on this. I have modified the Help text to reflect this.